### DISCUSSION TITLE
Not able to install GitHub copilot on code server.

### DISCUSSION THREAD

**Discussion by beeralas** [coder/code-server#6456] [Q&A]
Is code server support GitHub copilot?                 We are not able to install on our environment. We installed the latest version of code server. 
![16959787730125646121331257797111](https://github.com/coder/code-server/assets/27140096/063ff83b-22f8-4830-b03f-714531048862)
![16959788164873345239672290129994](https://github.com/coder/code-server/assets/27140096/5d034f1a-a50b-425e-a298-c9c7b4c91210)
![Uploading 16959788484725407745353665662010.jpg…]()

**Comment by code-asher** [comment]
4.17.1 should work with Copilot, but as far as installation goes, code-server uses https://open-vsx.org as the extension marketplace and Copilot is not available there because it is closed source and the authors have not submitted it to Open VSX.

The only way to install would be to download the .vsix file from Microsoft's marketplace and install it manually into code-server but unfortunately their terms of service appear to prohibit installing marketplace assets into non-Microsoft distributions of VS Code.
